### PR TITLE
Code examples for different sensealgs

### PR DESCRIPTION
**Pull request by frankschae**
Took a LV ODE with a linear time dependence in the interaction term:

dx = α * x - β * x * y * t 
dy = -δ * y + γ * x * y * t

Implements:
- an Euler solver
- discrete-forward, discrete-adjoint, continuous-forward, continuous-adjoint (most naive backsolve adjoint) with the Euler solver

### CODE REVIEW

File: code/sensitivity/ODEs.jl [hunk 1]
```diff
@@ -0,0 +1,197 @@
+# free fall ODE
```
Comment (frankschae) [inline code/sensitivity/ODEs.jl:1]:
```suggestion
# Lotka Volterra with time-dependent interaction
```

**Comment by facusapienza21** [comment]
This looks great @frankschae ! I will try to run the example today and see if there is something else we can do here!

**Comment by facusapienza21** [comment]
@frankschae what do you think is the best way to handle environments here? I noticed that you are adding a new `Project.toml` in the `sensitivity` folder instead of adding dependencies in the existing `Project.toml`. I am trying to think if it is better to have different small environment for the different examples or just a global one. Any thoughts on this?

**Comment by frankschae** [comment]
oh I didn't see the other Project.toml file  -- I think one is fine if we don't have too many dependencies. Several Project.toml files for different examples have the advantage that it will be easier to add new examples and getting started, for the first time, with individual examples is faster (with the obvious trade-off that if someone wants to run all examples he has to instantiate/precompile all of these individually).

**Comment by facusapienza21** [comment]
Ok! I will accept the PR as it is and move the required dependencies to the existing `Project.toml`. As long as we don't have any dependency issue, we should be fine, specially if we want to have very minimalistic scrips.

**Comment by frankschae** [comment]
I can also quickly update the PR -- whatever is more convenient for you :-)

**Comment by facusapienza21** [comment]
Oh that will be better! Thank you @frankschae ! I will merge the PR once you had made the change in the `Project.toml`.

**Pull request merged**